### PR TITLE
Fixes #7044 - Cloning host via hammer

### PR DESCRIPTION
**Pull request by dhlavac**
Allows clone hosts via hammer. For testing need to generate API cache from PR -> https://github.com/theforeman/foreman/pull/5940

### CODE REVIEW

File: lib/hammer_cli_foreman/host.rb [hunk 1]
```diff
@@ -309,6 +309,20 @@ class UpdateCommand < HammerCLIForeman::UpdateCommand
       include HammerCLIForeman::Hosts::CommonUpdateHelp
     end
 
+    class CloneCommand < HammerCLIForeman::UpdateCommand
+      action :clone
+
+      command_name 'clone'
+      option "--id", "ID", _("ID of host to be cloned"), :required => true
```
[the remark this answers is not in the document]
Comment (dhlavac) [reply lib/hammer_cli_foreman/host.rb]: Hi 
1. Which new-name ?
2. You mean cloning host by name ?
3. Descriptions changed

[the remark this answers is not in the document]
Comment (shiramax) [reply lib/hammer_cli_foreman/host.rb]: 1. you can see new-name in : hammer host clone --help under Options.
mshira@dhcp-2-122:~/git/hammer-cli-foreman(clone2)$ hammer host clone --help
Usage:
    hammer host clone [OPTIONS]

Options:
 --ask-root-password ASK_ROOT_PW        One of true/false, yes/no, 1/0
 --compute-attributes COMPUTE_ATTRS     Compute resource attributes
                                        Comma-separated list of key=value
 --id ID                                ID of host to be cloned
 --interface INTERFACE                  Interface parameters
                                        Comma-separated list of key=value
                                        Can be specified multiple times.
 --ip IP                                IP of the new cloned host
 --mac MAC                              MAC address of the new cloned host
 --name NAME                            Name of the new cloned host
 --new-name NEW_NAME         
^^^  

BTW, now that you added all the compute resource options,  you have couple of values that appear twice: 
for example: 
mac, ip and name are presented twice, under "Options" and under "Available keys for --interface"

**Comment by dhlavac** [comment]
@shiramax option -new-name removed. Multiple options for mac and ip is because you clone only primary interface (and specify ip amd mac for it) and also you can add more interfaces and specify it in --interface option.

**Comment by shiramax** [comment]
[test hammer]

**Comment by shiramax** [comment]
The foreman PR that is creating the API of clone hosts was closed, So I'm closing this one as well, please feel free to reopen if you wish to continue working on it.

**Pull request closed without merging**